This file and its contents are supplied under the terms of the
Common Development and Distribution License ("CDDL"), version 1.0.
You may only use this file in accordance with the terms of version
1.0 of the CDDL.

A full copy of the text of the CDDL should have accompanied this
source.  A copy of the CDDL is also available via the Internet at
http://www.illumos.org/license/CDDL.

Copyright (c) 2018, Joyent, Inc.

---

Files in this directory are used by kernel and user space code.  Things that are
unsupported in the kernel need to be excluded for kernel builds.


diff --git a/usr/src/common/acpica/disassembler/dmbuffer.c b/usr/src/common/acpica/disassembler/dmbuffer.c
index 79c37e2..923a485 100644
--- a/usr/src/common/acpica/disassembler/dmbuffer.c
+++ b/usr/src/common/acpica/disassembler/dmbuffer.c
@@ -375,6 +375,7 @@ AcpiDmUuid (
         Data[8], Data[9],
         Data[10], Data[11], Data[12], Data[13], Data[14], Data[15]);
 
+#ifdef ACPI_APPLICATION
     /* Dump the UUID description string if available */
 
     Description = AcpiAhMatchUuid (Data);
@@ -382,6 +383,7 @@ AcpiDmUuid (
     {
         AcpiOsPrintf (" /* %s */", Description);
     }
+#endif
 }
 
 
diff --git a/usr/src/common/acpica/disassembler/dmresrcl2.c b/usr/src/common/acpica/disassembler/dmresrcl2.c
index c725bec..2e32731 100644
--- a/usr/src/common/acpica/disassembler/dmresrcl2.c
+++ b/usr/src/common/acpica/disassembler/dmresrcl2.c
@@ -250,8 +250,10 @@ AcpiDmGpioCommon (
     AcpiDmIndent (Level + 1);
     AcpiOsPrintf ("}\n");
 
+#ifdef ACPI_APPLICATION
     MpSaveGpioInfo (Info->MappingOp, Resource,
         PinCount, PinList, DeviceName);
+#endif
 }
 
 
@@ -541,7 +543,9 @@ AcpiDmI2cSerialBusDescriptor (
     AcpiDmDumpSerialBusVendorData (Resource, Level);
     AcpiOsPrintf (")\n");
 
+#ifdef ACPI_APPLICATION
     MpSaveSerialInfo (Info->MappingOp, Resource, DeviceName);
+#endif
 }
 
 
@@ -624,7 +628,9 @@ AcpiDmSpiSerialBusDescriptor (
     AcpiDmDumpSerialBusVendorData (Resource, Level);
     AcpiOsPrintf (")\n");
 
+#ifdef ACPI_APPLICATION
     MpSaveSerialInfo (Info->MappingOp, Resource, DeviceName);
+#endif
 }
 
 
@@ -710,7 +716,9 @@ AcpiDmUartSerialBusDescriptor (
     AcpiDmDumpSerialBusVendorData (Resource, Level);
     AcpiOsPrintf (")\n");
 
+#ifdef ACPI_APPLICATION
     MpSaveSerialInfo (Info->MappingOp, Resource, DeviceName);
+#endif
 }
 
 
